[Backport v4-5-branch] - Altis dashboard pagination wrapping.

robindevitt opened the pull request:
Backporting #372

roborourke commented:
Weird 🤔 Revert this backport for now then

roborourke commented:
Ahhh ok, I think you needed to finish the cherry picking using `git cherry-pick —continue` after resolving the first conflict

robindevitt commented:
looking at it a little closer now, there appears to be a conflict which is then causing the cherry pick to fail:
```
robindevitt@Robins-MacBook-Pro backport % git cherry-pick 8798a4a8542b2ce205441d776029ea25a2c5876f 6a5c920bf39f8cceb85a6ffebcc743b87d6c2b3e e42925de5a2cfb9a535fd43eed9cd8ae878537c2 9e1788d4f03f7b2442023895d52b37728d44c84f b45648a3442bb070086c41cf238ce65b74f3e3c5 98f5026fd7d4906cd8cf0a734b45518e9df388ae f98211e1b8ecf337d82284a6032867ce0fefb1cb b8b15ded077556d4ad02de74312b0e160ff94301
[v4-5-branch 808f3a3] Altis dashboard pagination wrapping.
 Date: Thu Apr 7 16:13:18 2022 +0200
 1 file changed, 2 insertions(+), 1 deletion(-)
[v4-5-branch 136deca] Restore the button margin change.
 Date: Thu Apr 7 16:20:09 2022 +0200
 1 file changed, 1 insertion(+), 1 deletion(-)
Auto-merging package-lock.json
CONFLICT (content): Merge conflict in package-lock.json
error: could not apply e42925d... Added new npm package for pagination componenet and implemented the new componenet.
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
```

roborourke commented:
Yeah that’s expected, it gives you instructions in that output it’s just a bit hard to read. You have to fix the conflict, git add then git commit, then continue the cherry pick. Rinse and repeat until the conflicts are resolved and it’s gone through all the commits
